modules/puzzles/util.py

```diff
@@ -77,3 +78,31 @@ def remove_duplicate_locations(locations):
             results.append(l)
             storedLocations.append(normalizedL)
     return results
+
+
+def brute_force_coordinates(digits_list):
+    # Add token to seperate latitude and longitude later
+    digits_list.append(',')
```

Comment:
Is dit de goeie aanpak? Dit maakt het wel significant trager denk ik, misschien een goed idee om de comma achteraf ergens halverwege te plaatsen, want beginnen of eindigen met de comma maakt natuurlijk geen sense.

Reply:
Dat klopt. Ik heb alleen nog geen elegante manier gevonden om dit te doen. Ik zou dan namelijk van de verkregen permutaties nogmaals permutaties willen hebben waar de komma op enkele plekken kan staan per permutatie. De verschillende groepjes getallen mogen niet gescheiden worden en de komma moet tussen positie 6 en een positie afhankelijk van de totale lengte van de getallen staan. Daar komt nog bij dat het bepalen van de positie van de komma wat lastig te bepalen is omdat de permutatie bestaat uit tuples van een varierend aantal getallen. Suggesties zijn welkom :)

Reply:
Goed punt, ik denk er even over na